Trading/Exchange/binance.py

```diff
@@ -84,21 +105,60 @@ def set_sandbox_mode(self, is_sandboxed):
             {"symbol": self.get_exchange_pair(symbol),
              "marginType": self.BINANCE_MARGIN_TYPE_ISOLATED if isolated else self.BINANCE_MARGIN_TYPE_CROSSED})
 
-    def _cleanup_position_dict(self, position):
+    def cleanup_position_dict(self, position_dict):
         try:
             # If exchange has not position id -> global position foreach symbol
-            if ExchangeConstantsOrderColumns.ID.value not in position:
-                position[ExchangeConstantsOrderColumns.ID.value] = position[
+            if ExchangeConstantsOrderColumns.ID.value not in position_dict:
+                position_dict[ExchangeConstantsOrderColumns.ID.value] = position_dict[
                     ExchangeConstantsOrderColumns.SYMBOL.value]
-            if ExchangeConstantsPositionColumns.QUANTITY.value not in position \
-                    and self.BINANCE_FUTURE_QUANTITY in position:
-                position[ExchangeConstantsPositionColumns.QUANTITY.value] = position[self.BINANCE_FUTURE_QUANTITY]
-            if ExchangeConstantsPositionColumns.TIMESTAMP.value not in position:
-                position[ExchangeConstantsPositionColumns.TIMESTAMP.value] = 0  # TODO
-            if ExchangeConstantsPositionColumns.UNREALISED_PNL.value not in position \
-                    and self.BINANCE_FUTURE_UNREALIZED_PNL in position:
-                position[ExchangeConstantsPositionColumns.UNREALISED_PNL.value] = position[
+            if ExchangeConstantsPositionColumns.QUANTITY.value not in position_dict \
+                    and self.BINANCE_FUTURE_QUANTITY in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.QUANTITY.value] = position_dict[self.BINANCE_FUTURE_QUANTITY]
+            if ExchangeConstantsPositionColumns.TIMESTAMP.value not in position_dict:
+                position_dict[ExchangeConstantsPositionColumns.TIMESTAMP.value] = time.time()  # TODO
```

Comment:
Should we keep it like this ? If yes, shouldn't we return a millisec timestamp instead of a sec timestamp to be consistent with binance ?

Reply:
I added a `TODO` to remember that we are manipulating response data. I don't think we should return a millisec timestamp here because we'll convert it to a sec timestamp just after to uniformize with ccxt data.

Reply:
If it's right after for other exchanges then ok but otherwise if we have ms timestamps averywhere when using binance except here, I think it can complicate stuff, what do you think  about it ?